src/main/java/com/mkopec/ticketbookingapp/domain/RoomSeat.java

```diff
@@ -0,0 +1,29 @@
+package com.mkopec.ticketbookingapp.domain;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+import static javax.persistence.FetchType.LAZY;
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Data
+public class RoomSeat {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+
+    private Long id;
+
+    @ManyToOne(fetch = LAZY)
+    @JoinColumn(name = "seat_id")
+    private Seat seat;
+
+    @ManyToOne(fetch = LAZY)
+    @JoinColumn(name = "room_id")
+    private Room room;
+
+    @Column(name = "is_on_edge")
+    private Boolean isOnEdge;
```

Comment:
ciekawa koncepcja. A bez tego by się dało? I bez dodatkowej tabelki złączeniowej?

Reply:
Przyjąłem dodatkowe założenie, że miejsce na skraju może być zawsze zarezerwowane( Przy samym założeniu, że nie można zostawić jednego miejsca wolnego byłby problem z rezerwacją tego miejsca na skraju [ REZERWOWANE, WOLNE,  ZAJĘTE, .... ]. Wtedy nie musimy sprawdzać sąsiednich miejsc. To pozwala również na układ sali taki, że np. na środku sali może być przejście, wtedy mamy kolejne miejsca na skraju. ) Stąd ta zmienna isOnEdge. Z kolei obeszło by się bez tabelki złączeniowej, po prostu wrzucić numer rzędu i siedzenia do tej klasy.